server/vcs/markdown/markdown_renderer.go

```diff
@@ -86,6 +86,20 @@ func (m *Renderer) Render(res command.Result, cmdName command.Name, baseRepo mod
 	return m.renderProjectResults(res.ProjectResults, common, baseRepo)
 }
 
+// RenderProject formats the data into a markdown string for a project
+// This method relies on the underlying methods used by Render() so skipping tests since it's indirectly tested using the existing tests for Render()
```

Comment:
I think we should still have tests, it should be easy to just copy what we have for Render.

Reply:
Gotcha, I have copied and modified the tests where necessary!